mp/src/game/client/momentum/mom_map_cache.cpp

```diff
@@ -374,6 +375,23 @@ void CMapCache::SetMapGamemode(const char *pMapName /* = nullptr*/)
     if (!pMapName)
         pMapName = MapName();
 
+    const auto pGameModeOverride = g_pGameModeSystem->GetGameMode(mom_gamemode_override.GetInt())->GetType();
```

Comment:
The `p` type for hungarian notation usually signifies a pointer type, which this is not. `e` for enum could work better here.
```suggestion
    const auto eGameModeOverride = g_pGameModeSystem->GetGameMode(mom_gamemode_override.GetInt())->GetType();
```